frontend/src/metabase-types/guards/forms.ts

```diff
@@ -0,0 +1,14 @@
+import type {
+  StandardFormFieldDefinition,
+  CustomFormFieldDefinition,
+  FormFieldDefinition,
+} from "metabase-types/forms";
+
+export function isCustomWidget(
```

Comment:
These forms are legacy as well. Maybe just leave it where it is because it's about to get deprecated?